src/main/java/org/reaktivity/specification/nukleus/CoreFunctions.java

```diff
@@ -36,6 +37,8 @@
     private static final ThreadLocal<StringFW.Builder> STRING_RW = ThreadLocal.withInitial(StringFW.Builder::new);
     private static final ThreadLocal<String16FW.Builder> STRING16_RW = ThreadLocal.withInitial(String16FW.Builder::new);
 
+    private static final String CHALLENGE_CAPABILITY = "challenge";
```

Comment:
Change this to an inner `enum` called `Capability`, where the constructor knows the bit position, like `CHALLENGE(0x01)` below, and also implement `public static int of(String name, String... names)` on the `enum` class that walks through all the names and `ORs` them into a common capability mask.